registration/tests/models.py

```diff
@@ -471,17 +474,324 @@ def test_resend_activation_email_expired_user(self):
         Test the scenario where user tries to resend activation code
         to the expired user's email
         """
-        new_user = RegistrationProfile.objects.create_inactive_user(
+        new_user = self.registration_profile.objects.create_inactive_user(
             site=Site.objects.get_current(), send_email=False, **self.user_info)
         new_user.date_joined -= datetime.timedelta(
             days=settings.ACCOUNT_ACTIVATION_DAYS + 1)
         new_user.save()
 
-        profile = RegistrationProfile.objects.get(user=new_user)
+        profile = self.registration_profile.objects.get(user=new_user)
         self.assertTrue(profile.activation_key_expired())
 
         self.assertFalse(RegistrationProfile.objects.resend_activation_mail(
             email=self.user_info['email'],
             site=Site.objects.get_current(),
         ))
         self.assertEqual(len(mail.outbox), 0)
+
+
+class SupervisedRegistrationModelTests(RegistrationModelTests):
+    """
+    Test the model and manager used in the admin_approval backend.
+
+    """
+
+    user_info = {'username': 'alice',
+                 'password': 'swordfish',
+                 'email': 'alice@example.com'}
+
+    registration_profile = SupervisedRegistrationProfile
+
+    def setUp(self):
```

Comment:
use https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.override_settings instead of explict changes for `setUp` and `tearDown`